docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -185,61 +181,87 @@ repo_gpgcheck=1
 </TabItem>
 </Tabs>
 
-2. Run `sudo yum update` to refresh your repo data cache. Optionally, you may wish to also install all available updates.
+2. Run `sudo yum update` to refresh the repo data cache.
 2. Run `sudo yum install ziti-edge-tunnel` to install the RPM.
-2. Place a copy of the enrollment token JWT file in the `/opt/openziti/etc/identities` directory.
-2. Set the filemode and owner so that members of group `ziti` will be able to read and delete the token file.
-
-  ```text
-  sudo chown -cR :ziti        /opt/openziti/etc/identities
-  sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
-  ```
-
 2. Enable and start the service
 
     ```text
     sudo systemctl enable --now ziti-edge-tunnel.service
     ```
 
-2. The process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+2. Add an Identity
 
-    ```text
-    sudo systemctl restart ziti-edge-tunnel.service
-    ```
+## Adding an Identity
+
+### Restart to Load All in Identity Directory
+
+The tunneler will load all identities in the `--identity-dir` directory. The default location for identities is is
+`/opt/openziti/etc/identities`. Add identity files to this directory by copying the identity file into the directory and
+setting permissions for group `ziti` so the tunneler service may delete JWT enrollment tokens and read and write the
```

Comment:
> setting permissions for group `ziti`

I got all the way down here before realizing at no place in this document is it spelled out that the installer will create a `ziti` identity/group. There are lots of words around the group or the identity and chmod'ing but nothing that tells me that the installer will make these things and explaining to me why and that it'll exist.

That should be a whole section

Reply:
> Add identity files to this directory by copying the identity file into the directory and setting permissions for group ziti so the tunneler service may delete JWT enrollment tokens and read and write the JSON identity configuration files.

That's one large sentence. Break it into at least two, please. I would simply state that the files need to be owned by the ziti user. In the section above that you'll add about the ziti group, you might tell the user to add their OWN id to that group so that this won't be a problem going forward (aka like docker does) or maybe the installer adds $USER to the group? I dunno, but all that should be made clear in here